src/components/grouped-character/grouped-character.pw.tsx

```diff
@@ -0,0 +1,496 @@
+
+      await expect(
+        getDataElementByValue(page, "input").locator("..")
+      ).toHaveCSS("min-height", height);
+    });
+  });
+
+  ([
+    [[1, 2, 3], "1234567", "1-23-456"],
+    [[5, 3, 1], "987654321", "98765-432-1"],
+    [[2, 4, 2], "123456789", "12-3456-78"],
+  ] as [GroupedCharacterProps["groups"], string, string][]).forEach(
+    ([group, inputValue, outputValue]) => {
+      test(`should use ${group} as a group and use ${inputValue} as input value to produce ${outputValue} output value`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<GroupedCharacterComponent groups={group} />);
+
+        const input = getDataElementByValue(page, "input");
+        await input.fill(inputValue);
+        await input.blur();
+
+        const value = await input.evaluate(
+          (element: HTMLInputElement) => element.value
+        );
+        expect(value).toEqual(outputValue);
+      });
+    }
+  );
+
+  ([
+    ["-", "123456", "12-34-56"],
+    ["?", "sage", "sa?ge"],
+    ["#", "tests", "te#st#s"],
+    ["@", "abcdef", "ab@cd@ef"],
+    ["$", "987654321", "98$76$543"],
+    ["%", "123456789", "12%34%567"],
+    ["^", "123456", "12^34^56"],
+    ["!", "987654321", "98!76!543"],
+    ["*", "12ab34cd", "12*ab*34c"],
+  ] as [GroupedCharacterProps["separator"], string, string][]).forEach(
+    ([separator, inputValue, outputValue]) => {
+      test(`should use ${separator} as a separator and use ${inputValue} as input value to produce ${outputValue} output value`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<GroupedCharacterComponent separator={separator} />);
+
+        const input = getDataElementByValue(page, "input");
+        await input.fill(inputValue);
+        await input.blur();
+
+        const value = await input.evaluate(
+          (element: HTMLInputElement) => element.value
+        );
+        expect(value).toEqual(outputValue);
+      });
+    }
+  );
+});
+
+test.describe("check GroupedCharacter input", () => {
+  specialCharacters.forEach((specificValue) => {
+    test(`should check label renders properly with ${specificValue} as specific value`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<GroupedCharacterComponent label={specificValue} />);
+
+      await expect(getDataElementByValue(page, "label")).toHaveText(
+        specificValue
+      );
+    });
+  });
+
+  specialCharacters.forEach((specificValue) => {
+    test(`should check fieldHelp renders properly with ${specificValue} specific value`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<GroupedCharacterComponent fieldHelp={specificValue} />);
+
+      await expect(getDataElementByValue(page, "help")).toHaveText(
+        specificValue
+      );
+    });
+  });
+
+  specialCharacters.forEach((specificValue) => {
+    test(`should check tooltip renders properly with ${specificValue}s specific values`, async ({
```

Comment:
Small typo here: 
should check tooltip renders properly with ${specificValue} **as** specific values